api/ServiceWorkerRegistration.json

```diff
@@ -652,13 +652,13 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "45"
+                "version_added": "48"
               },
               "chrome_android": {
-                "version_added": "45"
+                "version_added": "48"
               },
               "edge": {
-                "version_added": "≤79"
+                "version_added": "18"
```

Comment:
This matches `maxActions` which I also tested in Edge 17/18 using https://mdn-bcd-collector.appspot.com/tests/api/Notification/maxActions to confirm it was added in Edge 18.

Since we don't have the source we can't be certain they were added together, but it's very likely I would say.